### ISSUE TITLE
clayui.com: general accessibility issues

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5143]
Some icons without description like configuration or github
Clay logo should have `alt=""` since the next text is "clay" right now the reader reads "Clay Clay"

**Comment by marcoscv-work** [comment]
<img width="529" alt="image" src="https://user-images.githubusercontent.com/7963804/195853303-5ed0e002-2588-48ff-943b-13838b09048a.png">
